contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/gobblin/build.gradle

```diff
@@ -0,0 +1,48 @@
+apply plugin: 'com.github.johnrengelman.shadow'
+
+repositories {
+    jcenter()
+    maven {
+        // For: "org.pentaho:pentaho-aggdesigner-algorithm:5.1.3-jhyde"
+        // required by: "gobblin-runtime:0.10.0 > org.apache.hive:hive-exec:1.0.1"
+        url "http://repo.spring.io/plugins-release"
```

Comment:
Since you are now excluding hive-exec, do we still need this? Let's remove it if we can.